README.md

```diff
@@ -114,6 +116,22 @@ make (your_framework)
 For example, `pip install -e ".[flask]" && make flask` launches a simple flask webapp with vulnpy
 endpoints.
 
+To run with Contrast, install the agent (`pip install -U contrast-agent`) and set
+`VULNPY_USE_CONTRAST=true` before running your desired `make` command.
+
+#### Running with Contrast in Docker
+
+`vulnpy` provides a Dockerfile that is also preconfigured to enable Contrast Security's
+instrumentation. To run a `vulnpy` web server with Contrast enabled using Docker:
+
+1. Copy a `contrast_security.yaml` configuration file into the `vulnpy` root directory
+2. Build the image with `docker build -t vulnpy:latest .` from the `vulnpy` root
```

Comment:
I was going to suggest the more general command ` docker build .` but when I tested that I see no tag on the image. Does your have a repository name? would be good to have all info populated https://stackoverflow.com/questions/38457669/docker-build-with-dockerfile-but-the-image-has-no-name-or-tag

```
REPOSITORY                                               TAG                                        IMAGE ID            CREATED             SIZE
<none>                                                   <none>                                     87b7a23593b4        16 seconds ago      1.24GB
```
also dang didn't expect that size!

Reply:
I don't think you can provide a default name or tag for an image inside of the Dockerfile, so tagging it from the CLI at build time is required if we want to see a repo/tag. Another option would be to use docker-compose to manage this instead of the default CLI.

As far as image size, the best thing I can think of is to use a smaller python base image:
```
python              3.8                 b0358f6298cd        47 hours ago        882MB
python              3.8-alpine          64df5e2068e3        3 weeks ago         44.5MB
```
I'm not sure if I'm ready to voluntarily use alpine, based on all the trouble it's given us in the past :|

Reply:
@danilito19 I tried switching to alpine and it made the build time substantially slower, because some python packages needed to be built from source instead of using a pre-build distribution. However, I also tried using the `slim` python base image, which worked beautifully after some finagling. This cut the image size in half:
```
vulnpy              latest              3ce96d4732d9        10 minutes ago      625MB
```